test/src/test/java/org/apache/accumulo/test/util/CertUtilsTest.java

```diff
@@ -53,13 +53,15 @@ private CertUtils getUtils() {
   @Test
   public void createSelfSigned() throws Exception {
     CertUtils certUtils = getUtils();
-    File tempSubDir = tempDir.toPath().resolve(testName()).toFile();
-    assertTrue(tempSubDir.isDirectory() || tempSubDir.mkdir());
-    File keyStoreFile = tempSubDir.toPath().resolve("selfsigned.jks").toFile();
-    certUtils.createSelfSignedCert(keyStoreFile, "test", PASSWORD);
+    Path tempSubDir = tempDir.resolve(testName());
+    if (!Files.isDirectory(tempSubDir)) {
+      Files.createDirectories(tempSubDir);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier